packages/docs/site/i18n/gu/docusaurus-plugin-content-docs/current/main/contributing/documentation.md

```diff
@@ -0,0 +1,132 @@
+---
+title: દસ્તાવેજીકરણમાં યોગદાન
+slug: /contributing/documentation
+description: WordPress Playground દસ્તાવેજીકરણમાં યોગદાન કેવી રીતે કરવું — ઇશ્યૂ ખોલવાથી લઈને PR મોકલવા, બ્રાઉઝરમાં એડિટ કરવા અને લોકલ પ્રીવ્યૂ ચલાવવાના પગલાં.
+---
+
+# દસ્તાવેજીકરણમાં યોગદાન
+
+<!--
+# Documentation contributions
+-->
+
+[WordPress Playground ની દસ્તાવેજીકરણ સાઇટ](/) તમારા જેવા સ્વયંસેવકો દ્વારા સંચાલિત છે — અને તેમને તમારી મદદ ગમે છે।
+
+<!--
+WordPress Playground's documentation site is maintained by volunteers like you, who'd love your help.
+-->
+
+WordPress/wordpress-playground રેપોઝિટરીમાં દસ્તાવેજીકરણ સંબંધિત તમામ ઇશ્યૂઝને લેબલ આપવામાં આવે છે: [\[Type\] Documentation](https://github.com/WordPress/wordpress-playground/issues?q=is:issue+state:open+label:"[Type]+Documentation) અથવા [\[Type\] Developer Documentation](https://github.com/WordPress/wordpress-playground/issues?q=is:issue+state:open+label:"[Type]+Developer+Documentation)। ખુલ્લા ઇશ્યૂઝની યાદી જુઓ અને તમને યોગ્ય લાગતો ઇશ્યૂ પસંદ કરો. અથવા તમને લાગે કે વર્તમાન દસ્તાવેજીકરણમાં કંઈક ગૂમ છે, તો તમારી સલાહ પર ચર્ચા કરવા માટે નવો ઇશ્યૂ ખોલો: [WordPress/wordpress-playground](https://github.com/WordPress/wordpress-playground)।
```

Comment:
The content is different from the original content; you can suggest improvements to the English version.